_rules/SC1-2-5-video-audio-alternative.md

```diff
@@ -0,0 +1,155 @@
+---
+name: video with audio has an accessible alternative
+rule_type: composite
+description: |
+  This rule checks video elements with audio have audio description
+
+success_criterion:
+- 1.2.5 # Audio Description (Prerecorded)§
+
+atomic_rules:
+- SC1-2-video-audio-description
+- SC1-2-video-media-alternative
+- SC1-2-video-description-track
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible on the page](#visible-on-the-page), where the video contains audio.
+
+### Expectation
+
+For each test target, the outcome of at least one of the following rules is passed:
+
+- [video element audio described](SC1-2-video-audio-description)
+- [Video element description track](SC1-2-video-description-track)
+- [Video as a media alternative for text](SC1-2-video-media-alternative)
```

Comment:
I'm not sure about this. The exception is not in the Success Criterion but in the definition for "Synchronized Media". That is, if the *audio* is a media alternative, then it is not part of synchronized media. Since we are only looking at video with audio, if the audio is a media alternative then it would be video-only that is not in synchronized media. So we need to check that the audio is a media alternative rather than the video, I think.

Reply:
@jkodu This is done by the way. Thanks.